### PR TITLE
[ButtonBar] Add a Typography Themer.

### PR DESCRIPTION
**Pull request by jverkoey**
Pivotal story: https://www.pivotaltracker.com/n/projects/2150420/stories/156769836

simulator screen shot - iphone se - 2018-04-18 at 12 07 32

### CODE REVIEW

File: components/ButtonBar/src/TypographyThemer/MDCButtonBarTypographyThemer.m [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+#import "MDCButtonBarTypographyThemer.h"
+
+@implementation MDCButtonBarTypographyThemer
+
++ (void)applyTypographyScheme:(nonnull id<MDCTypographyScheming>)typographyScheme
+                   toButtonBar:(nonnull MDCButtonBar *)buttonBar {
+  NSUInteger maximumStateValue = UIControlStateNormal | UIControlStateSelected |
+      UIControlStateHighlighted | UIControlStateDisabled;
+  for (NSUInteger state = 0; state <= maximumStateValue; ++state) {
+    [buttonBar setButtonsTitleFont:nil forState:state];
```
Comment (ianegordon) [inline components/ButtonBar/src/TypographyThemer/MDCButtonBarTypographyThemer.m:26]: Future / Offline Discussion:
We likely need to follow this pattern of enumerating through all the states in many of our themers.
Does it make sense to expose a reset method for use by our themers?

**Review by ianegordon — APPROVED** [review]
LGTM, thanks @jverkoey !

**Comment by ianegordon** [comment]
Bazel seems unhappy :/

File: components/ButtonBar/src/TypographyThemer/MDCButtonBarTypographyThemer.m (hunk 1 above)
Comment (jverkoey) [reply components/ButtonBar/src/TypographyThemer/MDCButtonBarTypographyThemer.m:26]: It may - though I'm not sure where that should live.

**Review by romoore — DISMISSED** [review]
Not a real rejection. Please dismiss.

**Pull request merged**